contracts/TiramisuSavingsClub.sol

```diff
@@ -0,0 +1,160 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+
+/// @title Tiramisu Savings Club v1
+/// @author Jerrall, Luis, Mayuko, Prashant
+/// @notice This dapp has a single smart contract architecture. It manages state for all savings clubs in this single contract
+/// @notice v1 was developed during a short period of time (DappCamp) with the goal of being a MVP. See whitepaper for future roadmap ideas
+/// @dev Contraint #1: A user (address) can only belong to a single group
+/// @dev Contraint #2: Only one pending request for money can exist at a time per group
+contract TiramisuSavingsClub {
+    // Store a list of group ids (the indices of this array) and the group owner address (the value at that index)
+    address[] public groupOwners;
```

Comment:
Consider creating a struct for Group so that you can store all the relevant information like Group Id, group Balance, members in the group. It will make your code lot more readable.

Reply:
I just refactored the implementation to use this pattern. You're right this cleaned things up considerably. Thanks!